### PR TITLE
Remove attribute setter method on exposure

### LINKED ISSUE #146: Deprecation message on rspec tests
Rails 5 has changed their ActionController::TestCase and it's raising an warning.

```ruby
DEPRECATION WARNING: ActionController::TestCase HTTP request methods will accept only
keyword arguments in future Rails versions.

Examples:

get :show, params: { id: 1 }, session: { user_id: 1 }
process :update, method: :post, params: { id: 1 }
 (called from block (3 levels) in <top (required)> at /Users/dev/hashrocket/decent_exposure/spec/integration_spec.rb:12)
```

If we fix the warnings adding the `params` key to scope the hash we will break `decent_exposure` to rails 4 versions

### ActionController::TestCase:

https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/test_case.rb#L643

screen shot 2016-05-10 at 4 37 58 pm

### LINKED ISSUE #157: Unable to expose question-mark methods
This is a regression of v. 3x

When exposing a

```ruby
expose(:valid?) { ... }
```

I got 

```
/home/ubuntu/hb/vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.5.2/lib/abstract_controller/helpers.rb:65:in `class_eval': /home/ubuntu/hb/vendor/bundle/ruby/2.2.0/gems/actionpack-4.2.5.2/lib/abstract_controller/helpers.rb:66: syntax error, unexpected '?', expecting ';' or '\n' (SyntaxError)
            def valid?=(*args, &blk)               ...
                     ^
```

### PR DESCRIPTION
**Pull request by vnegrisolo**
- [x] Remove attribute setters from view/helpers
- [x] Allow question-mark attributes to be exposed
- [x] remove warnings on rspec tests

closes #157
closes #146

### CODE REVIEW

File: spec/controller_spec.rb [hunk 1]
```diff
@@ -35,14 +35,19 @@ def params
       expect(controller).to respond_to(:thing)
     end
 
-    it "defines setter method" do
-      expect(controller).to respond_to(:thing=).with(1).argument
+    it "does not define setter method" do
+      expect(controller).to_not respond_to(:thing=)
     end
```
Comment (nashbridges) [inline spec/controller_spec.rb]: Setter in controller is OK, the problem is only with view setter.

File: lib/decent_exposure/exposure.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@ class Exposure
     attr_reader :controller, :options
 
     # Public: Initializes an Exposure and makes it accessible to a controller.
-    # For each Exposure, a getter and setter is defined.
-    # Those getters and setters are made available to
+    # For each Exposure, a getter is defined.
+    # Those getters are made available to
     # the controller as helper methods.
```
Comment (nashbridges) [inline lib/decent_exposure/exposure.rb]: For each Exposure, a getter and setter is defined.
Getter is made available to the controller as helper method.

File: lib/decent_exposure/exposure.rb [hunk 3]
```diff
@@ -50,7 +50,7 @@ def initialize(controller, name, fetch_block=nil, **options, &block)
       normalize_options
     end
 
-    # Public: Creates a getter and setter methods for the attribute.
+    # Public: Creates a getter method for the attribute.
     # Those methods are made avaiable to the controller as
```
Comment (nashbridges) [inline lib/decent_exposure/exposure.rb]: ditto

File: lib/decent_exposure/exposure.rb (hunk 2 above)
Comment (vnegrisolo) [inline lib/decent_exposure/exposure.rb]: fixed, thanks

**Pull request merged**